NEWS

```diff
@@ -6,6 +6,15 @@ Please send bug reports, questions and suggestions to
 Version 4.4.3
 * Fix the value of SUNMD5_MAX_ROUNDS.
 * Add generated C++-guards to <xcrypt.h>.
+* Add --enable-obsolete-api-enosys configure option.
+  If enabled, this option replaces the obsolete APIs (fcrypt,
+  encrypt{,_r}, and setkey{,_r}) with POSIX compliant stubs setting
+  "errno" to "ENOSYS", instead of actually performing any real operation.
+  The encrypt{,_r} function will - for security reasons - additionally
+  overwrite the data-block argument with random data.
```

Comment:
"For security reasons, the encrypt{,r} functions will also overwrite their data-block argument with random bits."

Reply:
Fixed.